File: logineer/src/main/java/br/com/gabryel/logineer/service/PhoneService.java
package br.com.gabryel.logineer.service;

import br.com.gabryel.logineer.dto.PhoneDto;
import br.com.gabryel.logineer.entities.Phone;
import br.com.gabryel.logineer.entities.User;

import java.util.List;

public interface PhoneService {
    Phone register(User user, PhoneDto phoneDto);

    List<Phone> getPhones(User user);
}

File: logineer/src/main/kotlin/br/com/gabryel/logineer/service/PhoneService.kt
package br.com.gabryel.logineer.service

import br.com.gabryel.logineer.dto.PhoneDto
import br.com.gabryel.logineer.entities.Phone
import br.com.gabryel.logineer.entities.User

interface PhoneService {
    fun register(user: User, phoneDto: PhoneDto): Phone

    fun getPhones(user: User): List<Phone>
}